examples/geometry/juelich_tests/test6.wkt

```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((0 4, 2 4, 4 4, 6 4, 7 4, 11 4, 11 0, 7 0, 6 0, 4 0, 2 0, 0 0, 0 4)))
```

Comment:
Please rewrite the polygon with just the 4 corner points.

Reply:
this room is a rectangle with the size(11 x 4).
the rooms evacuation_time / test1 / test4 are rectangles of the size (12 x 4).
I do not think we need to keep this geometry.

Reply:
Lets remove it